python/adbc_driver_snowflake/adbc_driver_snowflake/__init__.py

```diff
@@ -112,6 +112,9 @@ class StatementOptions(enum.Enum):
     #: Number of concurrent streams being prefetched for a result set.
     #: Defaults to 10.
     PREFETCH_CONCURRENCY = "adbc.snowflake.rpc.prefetch_concurrency"
+    #: An identifier for a query/queries that can be used to find the query in
+    #: the query history.  Use a blank string to unset the tag.
+    QUERY_TAG = "adbc.snowflake.statement.query_tag"
```

Comment:
did we ever update the docs to point out the difference between statement options and connection options etc.?

Reply:
I don't think so. In general I'd like to improve the docs...

Reply:
https://github.com/apache/arrow-adbc/issues/2493